docs/metrics/README.md

```diff
@@ -386,22 +388,15 @@ and the `MetricStreamConfiguration.CardinalityLimit` setting. Refer to this
 [doc](../../docs/metrics/customizing-the-sdk/README.md#changing-the-cardinality-limit-for-a-metric)
 for more information.
 
-Given a metric, once the cardinality limit is reached, any new measurement which
-cannot be independently aggregated because of the limit will be dropped or
-aggregated using the [overflow
+Given a metric, once the cardinality limit is reached, any new measurement
```

Comment:
Given a metric, once the cardinality limit is reached, any new measurement
that cannot be independently aggregated will be aggregated using the
[overflow
attribute](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute).

In versions prior to 1.10.0, the default behavior when cardinality limit gets hit was to *drop* the measurement. Users had the ability to opt-in to use overflow attribute instead, but this behavior is the default and the only allowed behavior starting 1.10.0

^Feel free to pick this wording.

Reply:
According to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute), what happens after the limit is reached is not "independently aggregated" in any case.

> An overflow attribute set is defined, containing a single attribute otel.metric.overflow having (boolean) value true, which is used to report a synthetic aggregation of the [Measurements](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#measurement) that could not be independently aggregated because of the limit.

With that established, the following sentences talked about how we handled it: previously (before 1.6.0) it's dropped, then since 1.6.0 we have optional experimental feature, then since 1.10.0 we'll have a new behavior.

Reply:
It's also correct. Just different arrangements. I'm OK with either one.

Reply:
Updated.